bokeh/resources.py

```diff
@@ -290,7 +290,7 @@ def _collect_external_resources(self, resource_attr):
                 external_resources.extend(external)
 
         # Return only unique external resources
-        return list(set(external_resources))
+        return list(external_resources)
```

Comment:
I know we want the de-duplication, but its more important that things actually load for now.